### PR TITLE
P2022-1383 Create new retro card -> P2022-2073 Handle the button in d…

### PR DESCRIPTION
**Pull request by kmaslowiec**
…ialog

### CODE REVIEW

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDialogFragment.kt [hunk 1]
```diff
@@ -31,9 +32,18 @@ class RetroDialogFragment : BottomSheetDialogFragment() {
 
         buttonListener()
 
+        binding.newRetroName.editText?.doOnTextChanged { text, _, _, _ ->
+            text?.let { handleAddButton(text.length in 5..149) }
```
Comment (Nataniel-Antosik) [inline app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDialogFragment.kt:36]: It is the correct section for these numbers?

File: app/src/main/res/layout/retro_dialog_fragment.xml [hunk 2]
```diff
@@ -32,6 +32,8 @@
                 android:id="@+id/new_retro_name_input"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:hint="@string/retro_card_input_hint"
```
Comment (Nataniel-Antosik) [inline app/src/main/res/layout/retro_dialog_fragment.xml:35]: Why hint is "Name"?

File: app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDialogFragment.kt (hunk 1 above)
Comment (kmaslowiec) [reply app/src/main/java/com/intive/patronage/retro/retro/presentation/view/RetroDialogFragment.kt:36]: This is a logic operation. I believe there is no need to put the numbers to the xml file

File: app/src/main/res/layout/retro_dialog_fragment.xml (hunk 2 above)
Comment (kmaslowiec) [reply app/src/main/res/layout/retro_dialog_fragment.xml:35]: That how it is in the design. 
image
But to be honest 'Feedback' would be better. @MarekMacko @BorysChajdas @catchanus ?

Comment (Nataniel-Antosik) [reply app/src/main/res/layout/retro_dialog_fragment.xml:35]: If it's like in design maybe let's leave "name" option

Comment (kmaslowiec) [reply app/src/main/res/layout/retro_dialog_fragment.xml:35]: Note is ok. Thank you :)

**Review by Nataniel-Antosik — APPROVED** [review]

**Pull request merged**